src/labels/CustomLabels.labels

```diff
@@ -9875,6 +9875,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Error shown when errors must be removed before something else can occur.</shortDescription>
         <value>Clear errors before proceeding.</value>
     </labels>
+    <labels>
+        <fullName>stgConcurrentBatchError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgConcurrentBatchError</shortDescription>
+        <value>There is a running job for {0} batch</value>
```

Comment:
@sam-knox , can you please verify the generic error message displayed when a batch is tried to be executed but another same batch is running:
- user clicks [Run Batch] on NPSP Settings page
- user starts a batch at Apex Anon Window at Developer Console 
- the batch starts as the regular scheduled jobs.

Currently we check concurrency for the legacy and enhanced RD batches, but this will probably be expanded to other batches if/where required.

There is `CRLP_BatchSkipped`:
> Today&apos;&apos;s scheduled {0} batch job was skipped because a previous {0} job was still running. The next batch job will run as scheduled. No action is required; this error is for information purposes only."

 but it works when the batch is to be scheduled. I am not sure the message is aligned when user clicks on [Run Batch] at NPSP Settings. Thoughts?

Reply:
Yeah, a very good point @RadSFDeveloper. Let me think on that for a bit and come up with language that works in all these scenarios.

Reply:
@RadSFDeveloper - Since this is going to be a generic error message, can we use the "common" identified here for the label name instead of "stg". Full name would be: "commonConcurrentBatchError"

Reply:
Thanks @sam-knox , is the message `<value>There is a running job for {0} batch</value>` is going to be changed or it is good as is?

Reply:
@RadSFDeveloper - That's going to change. Sorry, Melissa was just showing me how suggesting mode works on GitHub. I'll post the actual changes in my review.

Reply:
Giving the feedback in the comment is good as well, because (if I recall correctly) when the suggestion is accepted a new commit is created, meaning another build. If the code review changes are done locally, and committed once to the remote repo (regardless how many local commits), then build run will once.